tests/optional_scalars_generated.rs

```diff
@@ -629,15 +690,47 @@ impl std::fmt::Debug for ScalarStuff<'_> {
   }
 }
 #[inline]
+#[deprecated(since="1.13", note="Deprecated in favor of `root_as...` methods.")]
 pub fn get_root_as_scalar_stuff<'a>(buf: &'a [u8]) -> ScalarStuff<'a> {
-  flatbuffers::get_root::<ScalarStuff<'a>>(buf)
+  unsafe { flatbuffers::root_unchecked::<ScalarStuff<'a>>(buf) }
 }
 
 #[inline]
+#[deprecated(since="1.13", note="Deprecated in favor of `root_as...` methods.")]
 pub fn get_size_prefixed_root_as_scalar_stuff<'a>(buf: &'a [u8]) -> ScalarStuff<'a> {
-  flatbuffers::get_size_prefixed_root::<ScalarStuff<'a>>(buf)
+  unsafe { flatbuffers::size_prefixed_root_unchecked::<ScalarStuff<'a>>(buf) }
 }
 
+#[inline]
+pub fn root_as_scalar_stuff(buf: &[u8]) -> Result<ScalarStuff, flatbuffers::InvalidFlatbuffer> {
+  flatbuffers::root::<ScalarStuff>(buf)
+}
+#[inline]
+pub fn size_prefixed_root_as_scalar_stuff(buf: &[u8]) -> Result<ScalarStuff, flatbuffers::InvalidFlatbuffer> {
+  flatbuffers::size_prefixed_root::<ScalarStuff>(buf)
+}
+#[inline]
```

Comment:
@CasperN I'd like us to add auto-generated doc comments that say that the Verification system is experimental.

Reply:
Done